media/css/newsletter/fxnewsletter-subscribe.less

```diff
@@ -128,6 +128,7 @@
 
     #footer-email-errors {
         .span-all;
+        padding: (@baseLine / 2) 0;
```

Comment:
Suggest changing this to just `padding-bottom: @baseLine;` to avoid extra padding at the top too?